public/controllers/agent/components/register-agent.js

```diff
@@ -274,13 +272,17 @@ export const RegisterAgent = withErrorBoundary(
         this.state.selectedVersion === 'windows7'
       ) {
         return 'NET START WazuhSvc';
-      } else return '';
+      } else {
+        return '';
+      }
     }
 
     systemSelectorWazuhControlMacos() {
       if (this.state.selectedVersion == 'sierra') {
-        return 'sudo /Library/Ossec/bin/wazuh-control start';
-      } else return '';
+        return '/Library/Ossec/bin/wazuh-control start';
```

Comment:
**nitpick**: This command needs to be run with `sudo`. We should create another pull request or add the commit to fix this in the development branch 4.4 because this pull request was merged with the problem.

Reply:
Good catch